official/resnet/cifar10_main.py

```diff
@@ -189,7 +193,8 @@ def loss_filter_fn(name):
                                 learning_rate_fn=learning_rate_fn,
                                 momentum=0.9,
                                 data_format=params['data_format'],
-                                loss_filter_fn=loss_filter_fn)
+                                loss_filter_fn=loss_filter_fn,
+                                version=params['version'])
```

Comment:
Word of warning: this is going to conflict with #3472 . Won't be too hard to resolve, I think, but... be prepared.